dev-libs/libite/libite-2.4.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A collection of useful BSD APIs"
+HOMEPAGE="https://github.com/troglobit/libite"
+SRC_URI="https://github.com/troglobit/libite/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="static-libs"
+
+src_configure() {
+	econf --enable-static=$(usex static-libs)
```

Comment:
Since nothing depends on this package with static libs, can we disable them?
https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0302

Reply:
Sure, I'll just change:
```
```

Reply:
No no, get rid of the USE flag altogether. Then do `econf --enable-static=false`

Reply:
Thanks for the review. I have implemented the requested changes.